cpu/esp32/include/periph_cpu_esp32h2.h

```diff
@@ -0,0 +1,178 @@
+/*
+ * Copyright (C) 2025 Gunar Schorcht
+ *
+ * This file is subject to the terms and conditions of the GNU Lesser
+ * General Public License v2.1. See the file LICENSE in the top level
+ * directory for more details.
+ */
+
+#pragma once
+
+/**
+ * @ingroup     cpu_esp32
+ * @{
+ *
+ * @file
+ * @brief       ESP32-H2 specific peripheral configuration
+ *
+ * @author      Gunar Schorcht <gunar@schorcht.net>
+ */
+
+#include "sdkconfig.h"
+
```

Comment:
```suggestion
#include "macros/units.h"

```

To use the `MHZ()` macro.

Reply:
See the comment above.

Reply:
That's the error when `macros/units.h` is included here.
```
/home/gs/src/RIOT-Xtensa-ESP.esp-idf-5.4/build/pkg/esp32_sdk/components/esp_hw_support/port/esp32h2/include/soc/rtc.h:51:9: error: "MHZ" redefined [-Werror]
   51 | #define MHZ (1000000)
      |         ^~~
In file included from /home/gs/src/RIOT-Xtensa-ESP.esp-idf-5.4/cpu/esp32/include/periph_cpu_esp32h2.h:22,
                 from /home/gs/src/RIOT-Xtensa-ESP.esp-idf-5.4/cpu/esp32/include/periph_cpu.h:1041,
                 from /home/gs/src/RIOT-Xtensa-ESP.esp-idf-5.4/drivers/include/periph/gpio.h:80,
                 from /home/gs/src/RIOT-Xtensa-ESP.esp-idf-5.4/cpu/esp_common/include/gpio_arch_common.h:22,
                 from /home/gs/src/RIOT-Xtensa-ESP.esp-idf-5.4/cpu/esp32/include/gpio_arch.h:24,
                 from /home/gs/src/RIOT-Xtensa-ESP.esp-idf-5.4/cpu/esp32/periph/pm.c:23:
/home/gs/src/RIOT-Xtensa-ESP.esp-idf-5.4/core/lib/include/macros/units.h:48:9: note: this is the location of the previous definition
   48 | #define MHZ(x) (KHZ(x) * 1000UL)
      |         ^~~
cc1: all warnings being treated as errors

```

Reply:
How annoying. And they just have `one million` stored in that, so it's not even compatible with our macro... :/